### DISCUSSION TITLE
Internet and battery?

### DISCUSSION THREAD

**Discussion by Hellboy00000** [Hamza417/Positional#141] [General]
This app only use the internet for Google maps?
If i close the app it continue using the internet and wasting battery life in the foreground like this other app.

FOREGROUND_SERVICE - Allows the app to continuously receive GPS, magnetometer, and gyroscope readings,
even if the UI is killed by Android) https://www.ericbt.com/ebt_compass/privacypolicy

Thanks

**Comment by Hamza417** [comment]
I'm not sure what you mean, could you please elaborate?

Just for info, foreground service permission can't access the location in the background, you'd need to have background location permission for that. Positional doesn't do anything in the background, the foreground service is used to run the clock widget.

**Reply by Hellboy00000** [reply]
Just want to make sure that the app don't keep running anything when i close it.

**Reply by Hamza417** [reply]
It doesn't..
